src/components/App/App.js

```diff
@@ -30,8 +31,16 @@ class App extends Component {
             <Switch>
               <Route exact path ='/'
                 render={() => (
-                  this.state.error ?
-                   <h1>Sorry no movies found 🤷‍♂️</h1>
+
+                  !this.state.movies.length && !this.state.error ?
+                  <div className='loading'>
+                  <h1>LOADING</h1>
+                  <img src={tater} className="tater-logo" alt="logo" />
+                  </div>
+
+                  : this.state.error ?
+                   <h1>Sorry, no movies found</h1>
+
```

Comment:
Glad we added the loading icon to the main page which takes longer to load all the movies than the details page took to load one movie and its videos.